kythe/cxx/indexer/cxx/testdata/rec/rec_agg_init.cc

```diff
@@ -38,6 +54,9 @@ void fn(T&&...);
 void f() {
   //- @S ref StructS
   //- @"1" ref/init FieldA
+  //- !{ @"}" ref/init FieldB }
+  //- !{ @"}" ref/init FieldC }
+  //- !{ @"}" ref/init FieldD }
   auto s = S{1};
 
   //- @T ref StructT
```

Comment:
Is it worth adding some negative assertions on T as well, or is that sufficiently covered by those in f() above?

Reply:
That should be covered by the S (above) and V case (below).

> Is it worth adding some negative assertions on T as well, or is that sufficiently covered by those in f() above?

Those should be sufficiently covered by the S case (above) and V case (below).